assets/queries/serverlessFW/serverless_functions_with_full_privileges/metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "59ebb4f3-2a6c-46dc-b4f0-cc5418dcddcd",
+  "queryName": "Serverless Functions With Full Privileges",
```

Comment:
From what I understood, the query is not looking to serverless functions. I think we should improve the query  name and query description